src/data/articleTree.js

```diff
@@ -7,4 +7,8 @@ module.exports = {
   '06': ['artworks-page-test','pagina-opere-test'],
   '07': ['2019-02-01-my-first-gatsby-blog-post','2019-02-01-mio-primo-blog-post-gatsby'],
   '08': ['2019-03-11-news-from-art-planet', '2019-03-11-notizie-dal-pianeta-arte'],
+  '09': ['artworks/painting','opere/pittura'],
+  '10': ['artworks/sculpture','opere/scultura'],
+  '11': ['artworks/performance','opere/performance'],
+  '12': ['artworks/interactivity','opere/arte-interattiva'],
```

Comment:
i had to change the name `interattività` to `arte-interattiva` because causing a glitch in the switch mechanism, probably caused by the `à` char.